src/Form/AdminSettingsForm.php

```diff
@@ -206,9 +227,13 @@ private function buildFieldMappingsForm() {
     $field_mappings_settings = $this->config('acquia_lift.settings')->get('field_mappings');
     $field_names = $this->getTaxonomyTermFieldNames();
 
+    $taxonomy_vocabulary_url_text = t('Taxonomy Vocabulary');
+    $taxonomy_vocabulary_url = Url::fromRoute('entity.taxonomy_vocabulary.collection', [], ['attributes' => ['target' => '_blank']]);
+    $taxonomy_vocabulary_link = Link::fromTextAndUrl($taxonomy_vocabulary_url_text, $taxonomy_vocabulary_url)->toString();
+
     $form = [
       '#title' => t('Field Mappings'),
-      '#description' => t('Map Taxonomy vocabulary fields to "content section", "content keywords", and "persona" fields.'),
+      '#description' => t('Create ') . $taxonomy_vocabulary_link . t(' and map to "content section", "content keywords", and "persona" fields.'),
```

Comment:
This substitution should be done with placeholder values and arguments.  See info here: https://www.drupal.org/node/322732 with API info https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Render%21FormattableMarkup.php/function/FormattableMarkup%3A%3AplaceholderFormat/8.2.x and https://api.drupal.org/api/drupal/core%21includes%21bootstrap.inc/function/t/8.2.x